### PR TITLE
created main page

### PR DESCRIPTION
**Pull request by Ksenaaa**

### CODE REVIEW

File: src/components/Main/Benefits/constants/benefitsCards.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import cardImg from 'img/svg/main-card-icon.svg'
+import deliveryImg from 'img/svg/main-delivery-icon.svg'
+import homeImg from 'img/svg/main-home-icon.svg'
+import pigImg from 'img/svg/main-pig-icon.svg'
+
+export const benefitsCards = [
+  {
+    id: 1,
```
Comment (vell28) [inline src/components/Main/Benefits/constants/benefitsCards.ts]: you can us name as id if it unique

File: src/components/Main/NewArrivals/NewArrivals.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  const { swipe, onSwipeLeft, onSwipeRight } = useSwipe()
+
+  const { windowWidth } = useSizeListener()
+
+  return (
+    <div className={styles.wrapper}>
+      <Link href={`/${routes.NEW_ARRIVALS}`} className={styles.title}>
+        New Arrivals
+      </Link>
+      <div className={styles.swipeWrapper}>
```
Comment (vell28) [inline src/components/Main/NewArrivals/NewArrivals.tsx]: it must be inside Slider

File: src/components/Slider/Slider.tsx [hunk 3]
```diff
@@ -1,54 +1,61 @@
...
-  , [sliderPictures?.length])
+  useEffect(() => {
+    if (!picturesLength) return
 
-  const handlerClickPrevious = useCallback(() =>
-    sliderPictures?.length && setVisibleSlide(preState => preState === 0 ? sliderPictures.length - 1 : preState - 1)
-  , [sliderPictures?.length])
+    if (swipe === sliderSwipe.LEFT) {
+      setVisibleSlide(preState => preState === 0 ? picturesLength - 1 : preState - 1)
+    } else if (swipe === sliderSwipe.RIGHT) {
```
Comment (vell28) [inline src/components/Slider/Slider.tsx]: else

File: src/components/Slider/Slider.tsx [hunk 4]
```diff
@@ -1,54 +1,61 @@
...
-  const handlerClickNext = useCallback(() =>
-    sliderPictures?.length && setVisibleSlide(preState => sliderPictures.length - 1 === preState ? 0 : preState + 1)
-  , [sliderPictures?.length])
+  useEffect(() => {
+    if (!picturesLength) return
 
-  const handlerClickPrevious = useCallback(() =>
-    sliderPictures?.length && setVisibleSlide(preState => preState === 0 ? sliderPictures.length - 1 : preState - 1)
-  , [sliderPictures?.length])
+    if (swipe === sliderSwipe.LEFT) {
```
Comment (vell28) [inline src/components/Slider/Slider.tsx]: enum must start form UpperCase

File: src/interface/newsInterface.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
+export interface INews {
+  id: string,
+  name: string,
+  text: string,
+  pictures: {
```
Comment (vell28) [inline src/interface/newsInterface.ts]: Array<string>

File: src/pages/news/index.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  return (
+    <div className={styles.wrapper}>
+      <h1>News</h1>
+    </div>
+  )
+}
+
+export default News
+
+export const getStaticProps: GetStaticProps = async ({ locale }) => ({
```
Comment (vell28) [inline src/pages/news/index.tsx:19]: do really need this function?

File: src/components/Slider/Slider.tsx (hunk 4 above)
Comment (Ksenaaa) [reply src/components/Slider/Slider.tsx]: fixed

File: src/components/Slider/Slider.tsx (hunk 3 above)
Comment (Ksenaaa) [reply src/components/Slider/Slider.tsx]: fixed

File: src/components/Main/Benefits/constants/benefitsCards.ts (hunk 1 above)
Comment (Ksenaaa) [reply src/components/Main/Benefits/constants/benefitsCards.ts]: fixed

File: src/interface/newsInterface.ts (hunk 5 above)
Comment (Ksenaaa) [reply src/interface/newsInterface.ts]: fixed

File: src/pages/news/index.tsx (hunk 6 above)
Comment (Ksenaaa) [reply src/pages/news/index.tsx:19]: Yes. Without it there will be an error

File: src/components/Main/NewArrivals/NewArrivals.tsx (hunk 2 above)
Comment (Ksenaaa) [reply src/components/Main/NewArrivals/NewArrivals.tsx]: fixed

**Review by vell28 — APPROVED** [review]

**Pull request merged**